test/images/README.md

```diff
@@ -0,0 +1,129 @@
+# Building Kubernetes test images
+
+## Overview
+
+All the images found here are used in Kubernetes tests that ensures its features and functionality.
+The images are built and published as manifest lists, allowing multiarch and cross platoform support.
+
+## Prerequisites
+
+In order to build the docker test images, a Linux node is required. The node will require `make`
+and `docker (version 18.06.0 or newer)`. Manifest lists were introduced in 18.03.0, but 18.06.0
+is recommended in order to avoid certain issues.
+
+The node must be able to push the images to the desired container registry. Depending on the
+container registry, it might require a different authentication method. For dockerhub, this
+can be done by running the command:
+
+```bash
+    docker login -u your-awesome-username -p anAwesomerPassword
+```
+
+Windows Container images are not built by default, since they cannot be built on Linux. For
+that, a Windows node with Docker installed and configured for remote management is required.
+
+### Windows node setup
+
+In order to build the Windows container images, a node with Windows 10 or Windows Server 2019
+with the latest updates installed is required. The node will have to have Docker installed,
+preferably version 18.06.0 or newer.
```

Comment:
Can you be more explicit about why this version is preferred?

Reply:
Hm, there's no particular reason why it has to be 18.06.0 or newer on the Windows nodes. 18.06.0 or newer was chosen for consistency with the requirements on the Linux node (see L11).

Reply:
most build things are 19.03 by now